plugins/api/src/experimental_api_plugin_adaptor.cpp

```diff
@@ -35,8 +35,7 @@ namespace {
 
     auto to_json(bytesBuf_t* b)
     {
-        std::string foo{(char*)b->buf, (char*)b->buf+b->len};
-        return json::parse({(char*)b->buf, (char*)b->buf+b->len});
+        return json::parse(std::string(static_cast<char*>(b->buf), b->len));
```

Comment:
Does this copy the buffer? A better solution might be to use the other overload for parse that takes pointers.

Reply:
this is definitely worse than the solution you have in your PR.

i'll change it to match yours.

Reply:
done